po/it.po

```diff
@@ -242,925 +244,795 @@ msgid "Designing a Library"
 
-#: src/SUMMARY.md:261
+#: src/SUMMARY.md:262
 #, fuzzy
 msgid "Channels"
 msgstr "Canali"
 
-#: src/SUMMARY.md:262
+#: src/SUMMARY.md:263
 #, fuzzy
 msgid "Unbounded Channels"
 msgstr "Canali illimitati"
 
-#: src/SUMMARY.md:263
+#: src/SUMMARY.md:264
 #, fuzzy
 msgid "Bounded Channels"
 msgstr "Canali delimitati"
 
-#: src/SUMMARY.md:264
+#: src/SUMMARY.md:265
 #, fuzzy
-msgid "Shared State"
-msgstr "Stato condiviso"
+msgid "Send and Sync"
+msgstr "Invia e sincronizza"
 
 #: src/SUMMARY.md:265
 #, fuzzy
-msgid "Arc"
-msgstr "Arco"
+msgid "Send"
+msgstr "Inviare"
 
-#: src/SUMMARY.md:266
+#: src/SUMMARY.md:265
 #, fuzzy
-msgid "Mutex"
-msgstr "Mutex"
+msgid "Sync"
+msgstr "Sincronizza"
 
 #: src/SUMMARY.md:268
 #, fuzzy
-msgid "Send and Sync"
-msgstr "Invia e sincronizza"
+msgid "Examples"
+msgstr "Esempi"
 
-#: src/SUMMARY.md:268
+#: src/SUMMARY.md:269
 #, fuzzy
-msgid "Send"
-msgstr "Inviare"
+msgid "Shared State"
+msgstr "Stato condiviso"
 
-#: src/SUMMARY.md:268
+#: src/SUMMARY.md:270
 #, fuzzy
-msgid "Sync"
-msgstr "Sincronizza"
+msgid "Arc"
+msgstr "Arco"
 
 #: src/SUMMARY.md:271
 #, fuzzy
-msgid "Examples"
-msgstr "Esempi"
+msgid "Mutex"
+msgstr "Mutex"
 
-#: src/SUMMARY.md:273
+#: src/SUMMARY.md:274 src/SUMMARY.md:294
 #, fuzzy
 msgid "Dining Philosophers"
 msgstr "Filosofi a tavola"
 
-#: src/SUMMARY.md:274
+#: src/SUMMARY.md:275
 #, fuzzy
 msgid "Multi-threaded Link Checker"
 msgstr "Verifica link multi-thread"
 
-#: src/SUMMARY.md:276
+#: src/SUMMARY.md:277
 #, fuzzy
 msgid "Concurrency: Afternoon"
 msgstr "Giorno 1: Pomeriggio"
 
-#: src/SUMMARY.md:278
+#: src/SUMMARY.md:279
 #, fuzzy
 msgid "Async Basics"
 msgstr "Nozioni di base sull'asincronismo"
 
-#: src/SUMMARY.md:279
+#: src/SUMMARY.md:280
 #, fuzzy
 msgid "async/await"
 msgstr "asincrono/in attesa"
 
-#: src/SUMMARY.md:280
+#: src/SUMMARY.md:281
 #, fuzzy
 msgid "Futures"
 msgstr "Futuri"
 
-#: src/SUMMARY.md:281
+#: src/SUMMARY.md:282
 #, fuzzy
 msgid "Runtimes"
 msgstr "Garanzie all’Esecuzione"
 
-#: src/SUMMARY.md:282
+#: src/SUMMARY.md:283
 #, fuzzy
 msgid "Tokio"
 msgstr "Tokyo"
 
-#: src/SUMMARY.md:283
+#: src/SUMMARY.md:284
 #, fuzzy
 msgid "Tasks"
 msgstr "Compiti"
 
-#: src/SUMMARY.md:284
+#: src/SUMMARY.md:285
 #, fuzzy
 msgid "Async Channels"
 msgstr "Canali asincroni"
 
-#: src/SUMMARY.md:286
+#: src/SUMMARY.md:287
 #, fuzzy
 msgid "Join"
 msgstr "Giuntura"
 
-#: src/SUMMARY.md:287
+#: src/SUMMARY.md:288
 #, fuzzy
 msgid "Select"
 msgstr "Selezionare"
 
-#: src/SUMMARY.md:288
+#: src/SUMMARY.md:289
 #, fuzzy
 msgid "Pitfalls"
 msgstr "Insidie"
 
-#: src/SUMMARY.md:289
+#: src/SUMMARY.md:290
 #, fuzzy
 msgid "Blocking the Executor"
 msgstr "Blocco dell'esecutore"
 
-#: src/SUMMARY.md:290
+#: src/SUMMARY.md:291
 #, fuzzy
 msgid "Pin"
 msgstr "Spillo"
 
-#: src/SUMMARY.md:291
+#: src/SUMMARY.md:292
 #, fuzzy
 msgid "Async Traits"
 msgstr "Tratti asincroni"
 
 #: src/SUMMARY.md:295
 #, fuzzy
+msgid "Elevator Operations"
+msgstr "# Funzionamento dell'ascensore"
```

Comment:
Nit: is it correct to have the # prefix?

Reply:
Good catch, it should go away since it'll end up generating a Markdown heading.

The `fuzzy` marker saves it: it means that the translations here won't actually be used for anything. All fuzzy translations need proof-reading by a human (who will take away the marker).

Reply:
Yeah, this is an auto-generated (by GCP Translate API) that it's still `fuzzy`-marked, meaning it's not "active". 

I'm using the brilliant [Poedit](https://poedit.net/) for this, and it marks all the fuzzy translations clearly (in orange) to make sure I know what's final and what not.

![Screenshot 2023-05-17 at 21 08 18](https://github.com/google/comprehensive-rust/assets/114508/cf8b29ca-c295-443c-8537-d51ee54bdf60)